uint8 id
float32 supplyVoltage
uint32 systemTime
tag_t[] tags

